core/templates/pages/contributor-dashboard-admin-page/services/format-contributor-attributes.service.spec.ts

```diff
@@ -0,0 +1,199 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for format contribution attributes service
+ */
+
+import {FormatContributorAttributesService} from './format-contributor-attributes.service';
+import {
+  QuestionReviewerStats,
+  QuestionSubmitterStats,
+  TranslationReviewerStats,
+  TranslationSubmitterStats,
+} from '../contributor-dashboard-admin-summary.model';
+
+describe('Format contribution attributes service', () => {
+  let formatContributionAttributesService: FormatContributorAttributesService =
+    new FormatContributorAttributesService();
+
+  let translationSubmitterStats: TranslationSubmitterStats =
```

Comment:
I'd put each of these within their corresponding tests so that you can easily compare input and output.

Reply:
Done